releases/release-6.2.0.md

```diff
@@ -0,0 +1,450 @@
+
+    [User document](/system-variables.md#tiflash_fine_grained_shuffle_batch_size-new-in-v620) [#4631](https://github.com/pingcap/tiflash/issues/4631) @[guo-shaoge](https://github.com/guo-shaoge)
+
+* TiFlash supports a newer version of storage format
+
+    The new storage format relieves high CPU usage caused by GC in high-concurrency and heavy workload scenarios. This significantly reduces IO traffic of background tasks, thereby boosting stability under high concurrencies and heavy workloads. At the same time, space amplification and disk waste can be significantly reduced.
+
+    In TiDB v6.2.0, data is stored in the new storage format by default. Note that if TiFlash is upgraded from earlier versions to v6.2.0, you cannot perform in-place downgrade on TiFlash, because earlier TiFlash versions cannot recognize the new storage format.
+
+    For more information about upgrading TiFlash, see [TiFlash v6.2.0 Upgrade Guide](/tiflash-620-upgrade.md).
+
+    [User document](/tiflash/tiflash-configuration.md#configure-the-tiflashtoml-file) [#3594](https://github.com/pingcap/tiflash/issues/3594) @[JaySon-Huang](https://github.com/JaySon-Huang) @[jiaqizho](https://github.com/jiaqizho) @[lidezhu](https://github.com/lidezhu)
+
+* TiFlash optimizes data scanning performance in multiple concurrency scenarios (experimental)
+
+    TiFlash reduces duplicate reads of the same data by merging read operations of the same data, and optimizes the resource overhead in the case of multiple concurrent tasks to improve data scanning performance. It avoids the situation where the same data has to be read separately in each task, or even the same data may be read multiple times at the same time, if the same data is involved in multiple concurrent tasks.
+
+    [User document](/tiflash/tiflash-configuration.md#configure-the-tiflash-toml-file) [#5376](https://github.com/pingcap/tiflash/issues/5376) @[JinheLin](https://github.com/JinheLin)
+
+* TiFlash adds Fast Mode for data scanning to increase read and write speed by sacrificing data consistency (experimental)
+
+    TiDB introduces Fast Mode in v6.2.0. It supports skipping consistency checks to increase the speed significantly. The Fast Mode is suitable for scenarios that do not require high accuracy and consistency of data such as offline analysis tasks. Previously, to ensure data consistency, TiFlash needed to perform data consistency checks during the data scanning process to find the required data from multiple different versions (called Normal Mode).
+
+    When you upgrade from an earlier version to TiDB v6.2.0, the default mode is Normal Mode for all tables, which ensures data consistency. You can set the Fast Mode or Normal Mode for each table independently. If the table is set to Fast Mode in TiDB v6.2.0, the mode setting will be disabled when you downgrade to a lower version, but this does not affect the normal data read. In this case, it is equivalent to Normal Mode that offers strong consistency read.
```

Comment:
```suggestion
    When you upgrade from an earlier version to TiDB v6.2.0, FastScan is not enabled by default for all tables, which ensures data consistency. You can enable FastScan for each table independently. If the table is set to FastScan  in TiDB v6.2.0, it will be disabled when you downgrade to a lower version, but this does not affect the normal data read. In this case, it is equivalent to strong consistency read.
```

Reply:
extra space here
"If the table is set to FastScan ^^ in TiDB"